tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -125,6 +144,35 @@ HloInstruction *MaybeConstantFoldBias(HloInstruction *bias) {
 // We then guide the buffer assignment to alias the buffer of the custom call
 // and C.
 class GemmRewriterVisitor : public DfsHloRewriteVisitor {
+ private:
+  // Short forms for ops commonly used in pattern matching:
+  static constexpr auto bitcast = [](auto operand) {
+    return m::Bitcast(operand).WithOneUser();
+  };
+  static constexpr auto broadcast_bias = [](auto bias) {
+    return m::Broadcast(bias, m::Op()).WithOneUser();
+  };
+  static constexpr auto convert_from_bf16 = [](auto operand) {
+    return m::Convert(operand).WithOneUser();
+  };
+  static constexpr auto gemm = [](auto gemm) {
```

Comment:
This ought to have a name different than `gemm` which indicates that it is specifically a cublasLt gemm

Reply:
Good point. I changed `gemm` so that it matches either `kGemmCallTarget` or `kCublasLtMatmulCallTarget`, whereas `gemm_cublaslt` is matches `kCublasLtMatmulCallTarget`.